mozilla-mobile/android-components/components/feature/addons/src/main/res/values/strings.xml

```diff
@@ -110,6 +110,18 @@
     <string name="mozac_feature_addons_updater_notification_content" tools:ignore="PluralsCandidate">%1$d new permissions are required</string>
     <!-- Name of the "notification channel" used for displaying a notification for updating an add-on. See https://developer.android.com/training/notify-user/channels -->
     <string name="mozac_feature_addons_updater_notification_channel">Add-on updates</string>
+    <!-- Name of the "notification channel" used for displaying a notification for new supported add-ons. See https://developer.android.com/training/notify-user/channels -->
+    <string name="mozac_feature_addons_supported_checker_notification_channel">Supported add-ons checker</string>
+    <!-- The tile of the notification, this will be shown to the user when one newly supported add-on is available.-->
+    <string name="mozac_feature_addons_supported_checker_notification_title">New add-on available</string>
+    <!-- The tile of the notification, this will be shown to the user when more than one newly supported add-ons are available.-->
+    <string name="mozac_feature_addons_supported_checker_notification_title_plural">New add-ons available</string>
+    <!-- The content of the notification, this will be shown to the user when one newly supported add-on is available. %1$s is the add-on name and %2$s is the app name most of the case it will be Firefox. -->
+    <string name="mozac_feature_addons_supported_checker_notification_content_one">Add %1$s to %2$s</string>
+    <!-- The content of the notification, this will be shown to the user when two newly supported add-ons are available. %1$s is the first add-on name and %2$s is second and %3$s is the app name most of the cases it will be Firefox. -->
+    <string name="mozac_feature_addons_supported_checker_notification_content_two">Add %1$s and %2$s to %3$s</string>
+    <!-- The content of the notification, this will be shown to the user when more than two newly supported add-ons are available. %3$s is the app name most of the cases it will be Firefox. -->
```

Comment:
@Amejia481 
Can you fix this comment? The placeable is `%1$s`, not `%3$s`.

Also, could we fix up these comments a bit, since we're touching them?

> %1$s is the add-on name and %2$s is the app name most of the case it will be Firefox.

%1$s is the add-on name. %2$s is the app name (in most cases Firefox).

> %1$s is the first add-on name and %2$s is second and %3$s is the app name most of the cases it will be Firefox.

%1$s is the first add-on name. %2$s is the second add-on name. %3$s is the app name (in most cases Firefox).

> %3$s is the app name most of the cases it will be Firefox.

%1$s is the app name (in most cases Firefox).

Reply:
Sorry, I will update it!